### PR TITLE
feat: v1 -> v2 switcher for ethereum-operations

### PR DESCRIPTION
**Pull request by vgorkavenko**

### CODE REVIEW

File: ethereum-operations/src/common/constants.ts [hunk 1]
```diff
@@ -1,5 +1,20 @@
 import BigNumber from "bignumber.js";
 
+// COMMON ADDRESSES
+export const LIDO_APP_REPO_ADDRESS =
+  "0xF5Dc67E54FC96F993CD06073f71ca732C1E654B1";
+export const LIDO_ADDRESS = "0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84";
+
+export const LIDO_CONTRACT_VERSION_SET_EVENT = "";
```
Comment (dgusakov) [inline ethereum-operations/src/common/constants.ts:8]: Add actual signature

Comment (vgorkavenko) [reply ethereum-operations/src/common/constants.ts]: done

File: ethereum-operations/src/agent.ts [hunk 2]
```diff
@@ -145,6 +192,26 @@ const timeout = async (agent: SubAgent) =>
...
 const handleBlock: HandleBlock = async (
   blockEvent: BlockEvent
+): Promise<Finding[]> => {
+  if (
+    !isNewVersionInitialized &&
+    protocolVersionSwitchBlock != 0 &&
+    blockEvent.blockNumber > protocolVersionSwitchBlock
+  ) {
+    await _initialize(subAgents, blockEvent.blockNumber);
+    isNewVersionInitialized = true;
```
Comment (madlabman) [inline ethereum-operations/src/agent.ts:202]: Is set already inside _initialize

Comment (vgorkavenko) [reply ethereum-operations/src/agent.ts:202]: it's in public `initialize`, not `_initialize`

**Pull request merged**